protocol/avdl/keybase1/teams.avdl

```diff
@@ -255,6 +255,11 @@ protocol teams {
     // Whether this is an open team.
     boolean open;
     TeamRole openTeamJoinAs;
+
+    // If there is a TLF associated, it shows up here. The last writer wins
+    // here, and we are not concerned about historical associations that have been
+    // overwritten (for now). An empty TLFID is given by an empty string.
+    TLFID tlfID;
```

Comment:
Adding this field without busting the cache means that in 4 months a client might return 0 to KBFS for a team when it should return an existing TLFID. Seems like the kind of thing that could lead to an accidental tlf reset. Is there a reason this is safe?

Reply:
how could it, since its cached version of the sigchain would be stale?

Reply:
The snapshots will be "stale" but that just means merkle will be polled. Stale snapshots are not thrown out*. They are appended to or marked as fresh.
Right now your client is caching TeamData objects that, if the cache is not busted, will be read by this code when you update.

* if you transition from non-admin to admin and have stubbed links. Or something. But we may want to .